### ISSUE TITLE
use "HH:MM:SS" strings in place of time dicts in generated docs

### ISSUE THREAD

**Issue by adamgreig** [ukhas/habitat#177]
this affects payload_telemetry docs via the parser, and listener_telemetry docs via dl-fldigi and other trackers.

currently we use

```
"time": {"hour": 12, "minute": 34, "second: 46}
```

this would be neater as

```
"time": "12:34:46"
```

but, changing it might be quite sad. thoughts?

**Comment by danielrichman** [comment]
habitat:

#188 (pull request) Parse time to HH:MM:SS, not dict

dl-fldigi:

https://github.com/danielrichman/dl-fldigi/commit/05f7bab4a7d94dbdc8ae3e0da0268c968dc4110a

spacenearus uploader:

https://github.com/danielrichman/habitat-transition/commit/1eb8a5a1281c45e2ad14927eacfae4129709f6b8

TODO:

transition database.

**Comment by danielrichman** [comment]
Have just tested those changes to dl-fldigi & spacenearus uploader in a vm togther; looks fine

**Comment by danielrichman** [comment]
closed by #188
